presto-orc/pom.xml

```diff
@@ -179,6 +179,12 @@
             <artifactId>jcl-over-slf4j</artifactId>
             <scope>test</scope>
         </dependency>
+
+        <dependency>
```

Comment:
Is this dependency required ? We can just do.

```
File tempfile = File.createTempFile("some_prefix", "." + fileExtension);
                tempfile.deleteOnExit();
```

Reply:
> File tempfile = File.createTempFile("some_prefix", "." + fileExtension);
>                 tempfile.deleteOnExit();

@arunthirupathi Not aware we can do the same thing using standard library. Let me change the code removing the external dependency